test/files/neg/t8035-removed.scala

```diff
@@ -1,4 +1,4 @@
-// scalac: -Xfuture
+// scalac: -Xsource:3.0
```

Comment:
How about 2.14 instead of 3.0 ? We need to spread the pain a bit :).

Reply:
This represents a lot of pain, for not much gain (that I can see). Open to discuss, but my initial position is mild skepticism. I'm not yet convinced this should even be in 3.0

Reply:
Yes, I'm not sure anymore deprecation is the right thing to do.

Reply:
Anyway, I probably hold the minority opinion on this. The remaining discussion is to decide the version. We could do 2.14 if people think that's better.

Reply:
hmm, I'm surprised this is controversial at all, I would like to understand your thinking here. maybe we should discuss further in a more visible place? (https://github.com/scala/bug/issues/8035, or Discourse?)

Reply:
If we want to move this to 2.14, I think we'll need to vote for that at a SIP meeting.

Reply:
Since when is the committee concerned with the precise version number a deprecation happens in? It doesn't even get to specify the version in which a new feature will be implemented: https://docs.scala-lang.org/sips/sip-submission.html says
> the committee asks the compiler maintainers to indicate the earliest version of Scala that can include the language change.

Note: *asks* the maintainers, it doesn't *tell us*. And we're also talking about the *earliest* version. Since deprecation is kind of the dual, we'd be asked about the *latest* version something should be deprecated. However, there isn't even a SIP process for deprecation -- that usually happens together with the new feature that replaces it. That particular feature will be discussed as part of the Scala 3 sip.

If you think the SIP committee mandate should be expanded to include require precise versions in which we should implement/remove things, please put that on the agenda for November. I'll say now that I'm not in favor.